core/pom.xml

```diff
@@ -614,6 +614,12 @@ THE SOFTWARE.
       <artifactId>jzlib</artifactId>
       <version>1.1.3-kohsuke-1</version>
     </dependency>
+    <!-- Should be removed after splitting maven-builder-plugin see JENKINS-26462 -->
+    <dependency>
```

Comment:
Yes, that's the part I was indicating Oleg it is missing.

On Fri, Jul 1, 2016 at 5:09 PM, Jesse Glick <notifications@github.com>
wrote:

> In core/pom.xml
> <https://github.com/jenkinsci/jenkins/pull/2352#discussion_r69313614>:
>
> > @@ -614,6 +614,12 @@ THE SOFTWARE.
> >        <artifactId>jzlib</artifactId>
> >        <version>1.1.3-kohsuke-1</version>
> >      </dependency>
> > +    <!-- Should be removed after splitting maven-builder-plugin see JENKINS-26462 -->
> > +    <dependency>
>
> 🐛 for lack of conversion of ProjectWithMaven. This was after all the
> predicted main work for this PR. :-)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jenkinsci/jenkins/pull/2352/files/54042d889d81f6cca6be956b6d9d68e3faeec0f7#r69313614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQXNBBowT9_wtopjT6JrReGelMylV3bDks5qRS2RgaJpZM4Ifc5B>
> .
>